dev-python/pythondialog/pythondialog-3.4.0.ebuild

```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Same here. Once you introduce 3.5.1 this version can be dropped, unless something is explicitly depending on it.